MSAL/src/MSALPublicClientApplication.m

```diff
@@ -440,6 +440,11 @@ - (MSALAccount *)accountForUsername:(NSString *)username
                               error:(NSError * __autoreleasing *)error
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->

Reply:
Updated